DependencyInjection/Compiler/TwigExceptionPass.php

```diff
@@ -24,6 +25,16 @@
     public function process(ContainerBuilder $container)
     {
         if ($container->has('fos_rest.exception_listener') && $container->has('twig.exception_listener')) {
+
+            // wrap our exception listener around twig exception listener
+            $wrappedExceptionListener = $container->findDefinition('twig.exception_listener');
+            $wrappedExceptionListener->setPublic(false);
+            $wrappedExceptionListener->setTags(array());
+
+            $container->setDefinition('fos_rest.wrapped_exception_listener', $wrappedExceptionListener);
+            $container->findDefinition('fos_rest.exception_listener')->addArgument(
+                new Reference('fos_rest.wrapped_exception_listener')
+            );
             $container->removeDefinition('twig.exception_listener');
```

Comment:
This was introduced in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/908